### PR TITLE
#[UIC-1127]: Cypress login command password is not encrypted

### PR DESCRIPTION
**Pull request by ankursinghal2005**
Move encryption function into utils

Scenario:
Cypress e2e test cases is not working

Test Steps:
Try to run Cypress e2e test

Expected Result:
Cypress e2e should be passed

### CODE REVIEW

File: superset/assets/cypress/utils/common.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  var _k = CryptoJS.enc.Utf8.parse('qw34sd78fh67asb1');
+  var _i = CryptoJS.lib.WordArray.random(16);
+  var _e = CryptoJS.AES.encrypt(_m, _k, {
+      iv: _i
+  });
+  _e = _i.concat(_e.ciphertext).toString(CryptoJS.enc.Base64);
+
+  return _e;
+}
+
```
Comment (arpit-agarwal) [inline superset/assets/cypress/utils/common.js]: can we not import in cypress from the lib directly?

File: superset/assets/src/utils/common.js [hunk 2]
```diff
@@ -126,3 +128,14 @@ export function prepareCopyToClipboardTabularData(data) {
   }
   return result;
 }
+export function encryptStr(_m) {
```
Comment (arpit-agarwal) [inline superset/assets/src/utils/common.js]: can we rename it to `encryptText`

File: superset/assets/cypress/utils/common.js (hunk 1 above)
Comment (ankursinghal2005) [reply superset/assets/cypress/utils/common.js]: we don't have common libs shared between modules.

File: superset/assets/src/utils/common.js (hunk 2 above)
Comment (ankursinghal2005) [reply superset/assets/src/utils/common.js]: done

**Pull request merged**